### ISSUE TITLE
DBRX training

### ISSUE THREAD

**Issue by ewof** [axolotl-ai-cloud/axolotl#1447]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/OpenAccess-AI-Collective/axolotl/discussions/categories/ideas) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/OpenAccess-AI-Collective/axolotl/labels/enhancement) didn't find any similar feature requests.

### 🔖 Feature description

:DDDD
train dbrx model
peft and such would be nice

### ✔️ Solution

uh

### Acknowledgements

- [X] My issue title is concise, descriptive, and in title casing.
- [X] I have searched the existing issues to make sure this feature has not been requested yet.
- [X] I have provided enough information for the maintainers to understand and evaluate this request.

**Comment by Undi95** [comment]
I've done a mirror of the base model if needed => https://huggingface.co/Undi95/dbrx-base
Dunno if the people from here needed access to it, but if it can make things advance faster, no gatekeep on this repo
I would really like to toy some finetune on that big boy

**Comment by mtisz** [comment]
Is DBRX fine-tuning supported already?

**Comment by NanoCode012** [comment]
In progress here by winglian: #1462
